homeassistant/components/nibe_heatpump/__init__.py

```diff
@@ -329,18 +330,17 @@ def available(self) -> bool:
             self.coordinator.data or {}
         )
 
-    def _async_read_coil(self, coil: Coil):
+    def _async_read_coil(self, data: CoilData):
```

Comment:
Side note: Missing return value type annotation.

Reply:
Right. Will fix in followup.